pgbouncer/ci/start-docker.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
With the changes to the take tasks this should be easy to put into the rake file instead of here: this is a pretty simple docker setup